### PR TITLE
feat(networks): Add Goerli testnet

### PR DESCRIPTION
**Pull request by qbzzt**
Closing:  INT-613

### CODE REVIEW

**Comment by qbzzt** [comment]
Converted to draft because there is no mechanism I know to get ETH on Optimism Goerli, and without that it's useless.

File: src/docs/useful-tools/networks.md [hunk 1]
```diff
@@ -41,18 +41,55 @@ However, we _highly_ encourage you to set up a private RPC endpoint with any of
...
+For now, use the Optimism provided endpoints in the table.
+
+### Test ETH
+
+You can obtain Goerli test ETH either [here](https://goerlifaucet.com/) or [here](https://faucet.paradigm.xyz/).
+
+Our gateway still doesn't support Goerli, but if you transfer Goerli ETH to [0x636Af16bf2f682dD3109e60102b8E1A089FedAa8](https://goerli.etherscan.io/address/0x636Af16bf2f682dD3109e60102b8E1A089FedAa8), you will get it on Optimistic Goerli.
+
+
+## Optimism Kovan (old testnet)
```
[the remark this answers is not in the document]
Comment (qbzzt) [reply src/docs/useful-tools/networks.md:84]: You mean Kovan before Goerli, or the way to transfer ETH to Goerli?

File: src/docs/guides/testing.md [hunk 2]
```diff
@@ -13,14 +13,18 @@ This page shows some additional testing utilities that you might find useful.
...
 
-| Address | Symbol | Decimals |
-| - | - | -: |
-| [`0xC905388A3F73d796494b4D8D164844e166B0bA48`](https://kovan-optimistic.etherscan.io/address/0xC905388A3F73d796494b4D8D164844e166B0bA48) | OUT-1 | 0 |
-| [`0xA058b82EAB4966B1D9A10Fe0aC224FC34a69751F`](https://kovan-optimistic.etherscan.io/address/0xA058b82EAB4966B1D9A10Fe0aC224FC34a69751F) | OUT-2 | 0 |
-| [`0xA6dB0d5e8069f4027055Dbe165956218B64BA5C0`](https://kovan-optimistic.etherscan.io/address/0xa6db0d5e8069f4027055dbe165956218b64ba5c0) | OUT-3 | 0 |
-| [`0x5F6572727a825D1B8e1DFa9810d52101ed6d522a`](https://kovan-optimistic.etherscan.io/address/0x5F6572727a825D1B8e1DFa9810d52101ed6d522a) | OUT-4 | 0 |
-| [`0x5a1081FcaF5886845D5FE49fe0DE583EA15C4df3`](https://kovan-optimistic.etherscan.io/address/0x5a1081FcaF5886845D5FE49fe0DE583EA15C4df3) | OUT-5 | 0 |
-| [`0x010445A1bec4BD8e35e8c08Fbf46c05B4CD00100`](https://kovan-optimistic.etherscan.io/address/0x010445A1bec4BD8e35e8c08Fbf46c05B4CD00100) | Out-x.3 | 3 |
+| Address | Symbol | Decimals | Network |
```
[the remark this answers is not in the document]
Comment (qbzzt) [reply src/docs/guides/testing.md]: There is a Goerli token there. Or do you think we need to have five? I didn't see the need yet, I created a large number on Kovan for Uniswap, which we don't have on Goerli yet (wave P1).

Or should I make it clearer that it's Goerli? I can do that.

File: src/docs/useful-tools/networks.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nicca42) [reply src/docs/useful-tools/networks.md:84]: I meant moving it above the Goerli section, but it is actually fine like this.

File: src/docs/guides/smart-contract-devs.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ Depending on your set up you will need to set up a local testnet. All of the abo
...
+So you've written your contracts and are ready to deploy them to the testnet! First things first you are going to need some test ETH on Optimism Goerli. Head to the [faucet](https://faucet.paradigm.xyz/) for some test ETH on Goerli.
 
-Next you will need to connect to the Optimism Kovan network to deploy your contracts. You can find all the required network information as well as PRC endpoints in our [networks page](../useful-tools/networks.md#optimism-kovan-testnet). Additionally we have a helpful page of [debugging tools](../useful-tools/debugging.md) you can make use of while working out those final bugs.
+Next you will need to connect to the Optimism Kovan network to deploy your contracts. You can find all the required network information as well as PRC endpoints in our [networks page](../useful-tools/networks.md#optimism-goerli). Additionally we have a helpful page of [debugging tools](../useful-tools/debugging.md) you can make use of while working out those final bugs.
```
Comment (Nicca42) [inline src/docs/guides/smart-contract-devs.md:20]: > Next you will need to connect to the Optimism Kovan network to deploy your contracts.

Next you will need to connect to the Optimism test network to deploy your contracts.

File: src/docs/developers/build/testing-dapps.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ Most Ethereum development stacks include features that make testing easier, whic
 Therefore, it is a good idea to run the majority of tests, which do not rely on Optimism-specific features, in the development stack.
 It is a lot faster.
 
-Ideally you would want to be able to run some tests on Optimism (either a [local development environment](dev-node.md) or [a test network](../../useful-tools/networks.md#optimism-kovan-testnet)).
+Ideally you would want to be able to run some tests on Optimism (either a [local development environment](dev-node.md) or a test network (either [Kovan](../../useful-tools/networks.md#optimism-kovan-old-testnet) or [Goerli](../../useful-tools/networks.md#optimism-goerli)).
```
Comment (Nicca42) [inline src/docs/developers/build/testing-dapps.md:18]: Put Goerli first here 

> either Goerli or Kovan 

Also add: 

a test network (either [Goerli](../../useful-tools/networks.md#optimism-kovan-old-testnet) or [Kovan](../../useful-tools/networks.md#optimism-goerli)). Note that [Kovan is being deprecated]() and should not be used going forward. 

With a link to the article about the depreciation

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**